.github/workflows/nightly.yaml

```diff
@@ -32,9 +29,9 @@ jobs:
         uses: ./.github/actions/docker-build
         with:
           registry: ${{ env.REGISTRY }}
-          username: ${{ env.USERNAME }}
-          password: ${{ secrets[env.PASSWORD_KEY] }}
-          imageName: ${{ env.IMAGE_NAME }}
+          username: ${{ vars.QUAY_USERNAME }}
```

Comment:
still unclear where this vars.QUAY_USERNAME is defined but if you think this is correct we can merge it